docs/src/hal/tools.adoc

```diff
@@ -305,9 +305,11 @@ APP = halreport /tmp/halreport.txt
 
 The function addf-ordering can be important for servo loops where
 the sequence of the functions computed at each servo period is
-important.  Typically, the order is: read input pins, do the
-motion command-handler and motion-controller functions, perform
-pid calculations, and finally write output pins.
+important.  Typically, the order is:
+. read input pins,
+. do the motion command-handler and motion-controller functions,
+. perform pid calculations, and finally
+. write output pins.
```

Comment:
Do you had a list in mind here?

Reply:
An enumerated list, yes.

Reply:
But the output is no enumerated list unfortunately.

Reply:
Blank line was missing. Thanks! Fixed in my latest push.